core/src/main/java/discord4j/core/object/entity/GuildMessageChannel.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package discord4j.core.object.entity;
+
+import discord4j.core.object.trait.Categorizable;
+import discord4j.core.object.trait.Invitable;
+import discord4j.core.object.util.Snowflake;
+import discord4j.core.spec.WebhookCreateSpec;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+/** A Discord channel that is associated to a {@link Guild} and can utilize messages. */
+public interface GuildMessageChannel extends Categorizable, GuildChannel, Invitable, MessageChannel {
```

Comment:
Can invites be created for news channels? I wouldn't be surprised, just checking.

Reply:
Let's assume we can.

Reply:
https://github.com/discordapp/discord-api-docs/issues/881#issuecomment-471109730
From this comment you can tell that news channels in fact can have invites.